client/gutenberg/editor/api-middleware/index.js

```diff
@@ -39,9 +40,12 @@ export const wpcomProxyMiddleware = options => {
 	// bypassing the apiFetch call that uses window.fetch.
 	// This intentionally breaks the middleware chain.
 	return new Promise( ( resolve, reject ) => {
+		const { body, data } = options;
```

Comment:
```js
export const wPM = ( { body, data, ...options } ) => {

	wpcomProxyRequest( {
		...options,
		...( ( body || data ) && { body: body || data } ),
		apiNamespace: 'wp/v2',
	} )
```

we can get by without `omit` here

Reply:
fixed on c05950213c92321487dade71083fb88bcdb01d48.

Reply:
well this will still work but the fix changed the behavior technically. note the difference here…

```js
const f = ( { body, data, ...options } ) => options;
const g = options => {
  const { body, data } = options;

  return options;
}
```

in the first one you'll get an object _without_ the `body` and `data` properties but in the second they are still there. the second is still working probably because we are reassigning `body` and `data` immediately after accepting their raw form.

okay so now I'm seeing that the behavior is definitely different because we'll be including `data` when previously we weren't.

we can use the object spread to grab the rest of the options in the function declaration (as in the example I gave) _or_ we'll have to create a new variable for the rest.

```js
const { body, data, ...otherOptions } = options;

{
  ...otherOptions,
  ...( ( body…
```

Reply:
That's why initial implementation used `...omit( options, [ 'data', 'body' ] )`, although using the spread operator is a far more clever 🤓 

Thanks @gwwar for creating https://github.com/Automattic/wp-calypso/pull/27283 😸